### PR TITLE
Refactor Office Page

### PR DESCRIPTION
**Pull request by milljoniaer**
## New Office Page

This adds the new designed Office Page including the team section with all the pictures I found.

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain21751-featofficepage.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/ec0f3919-bdc2-4dff-8649-8f420041a9d3/details)

:clock1: Build time: **1m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 70   |
| Accessibility |   :green_heart: 92   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 71   |

:link: [View full report](https://build-ec0f3919-bdc2-4dff-8649-8f420041a9d3.gtsb.io/reports/lighthouse/index.html)

File: src/pages/office.tsx [hunk 1]
```diff
@@ -41,79 +73,172 @@ const OfficePage = ({ location }: OfficePageProps) => {
...
+            scelerisque nisl consectetur et. Morbi leo risus, porta ac
+            consectetur ac, vestibulum at eros. Donec sed odio dui. Integer
+            posuere erat a ante venenatis dapibus posuere velit aliquet. Aenean
+            eu leo quam. Pellentesque ornare sem lacinia quam venenatis
+            vestibulum.
+          </StyledSectionHeader>
+          <TeamWrapper>
+            {/* {TEAM.map((member) => {
+              return (
+                <Image description={member.name} textAlign="bottom">
```
Comment (feedm3) [inline src/pages/office.tsx]: Just a proactive comment, guessing this didn't work: If you map over an array, you need to set the `key` property to every element. 

```
<Image description={member.name} textAlign="bottom" key={member.name}>
```

It's important to use a unique identifier for that. Best from the object you pick directly. Using the index of a for loop is a bad practice and can lead to unexpected errors:

Bad:
```
{TEAM.map((member, index) => {
              return (
                <Image description={member.name} textAlign="bottom" key={index}>
```

**Comment by georgiee** [comment]
I have modified this PR ad-hoc:
+ Created a json file to hold the our team + images. The images are transformed by the source plugin (managed by `gatsby-transformer-json`)
+ Added a few text but it's really to less we need more content on that page.
+ Added a few fixes on the landingpage
+ updated the stories, which are not ideal anyway because we can't show much with all the gatsby images involved

I really, really want to rename the page into "About us" because that's where we are heading to. For this we can't place the office into the focus and update our text. Then we need to talk about how we work and think. One small part is the office life then. I really want to update that aspect so I stopped myself. Let's merge this PR and continue.

**Review by georgiee — APPROVED** [review]
let's merge

**Pull request merged**